### ISSUE TITLE
Annealing learning rate, training loss spike.

### ISSUE THREAD

**Issue by AyaLahlou** [NVIDIA/tacotron2#387]
When training on a male voice using the pretrained model, my training loss spiked after 9k iterations. 
![image](https://user-images.githubusercontent.com/50680354/87651973-8705cb80-c74b-11ea-9a6f-7e68b2b58c62.png) 
I read other issues advising to anneal the learning rate, does it apply to this case? Also is there a way I can anneal the learning rate without restarting the training over?

**Comment by youmebangbang** [comment]
Loss spiking at the start on default learning rate (1e-3) is normal, you just need to go back to the previous checkpoint before the spike. Your loss has leveled out anyway, so it is time to lower your rate. If you are using 1e-3 set it to 1e-4.
